Services/UICore/classes/class.ilCtrl.php

```diff
@@ -983,13 +876,15 @@ private function getPathNew($a_source_node, $a_target_node)
 			$temp_node.= $cid;
 			$path[] = $temp_node;
 		}
-//if ($this->getCmdClass() == "ilmailfoldergui") var_dump($path);
-//var_dump($path);
 		return $path;
 	}
 
 	/**
-	 * set target script name
+	 * Set target script name.
+	 *
+	 * This is in fact used in various places, but as `grep -r setTargetScript .`
+	 * reveals is always set to "ilias.php". The target script stuff seems to be
+	 * pointless.
```

Comment:
I was not sure if there is some non-obvious reason for its existence. Will remove...